src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,64 @@
+package duke;
+
+import duke.DukeException;
+import duke.task.Task;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+import java.lang.Boolean;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads list of tasks from file.
+     *
+     * @throws DukeException If file not found.
+     * @return List of tasks.
+     */
+    public ArrayList<Task> load() throws DukeException {
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        try {
+            File file = new File("duke.txt");
+            if (!file.exists()) {
+              throw new DukeException("File not found");
+            }
+            Scanner fileScanner = new Scanner(file);
+            while (fileScanner.hasNextLine()) {
+                String taskString = fileScanner.nextLine();
+                String[] data = taskString.split(",");
+                String type = data[0];
+                assert data[1].equals("true") || data[1].equals("false") : throw DukeException("Invalid file input");
+                Boolean status = Boolean.parseBoolean(data[1]);
+                String text = data[2];
+                if (data[0].equals("T")) {
```

Comment:
The if-else class of statements should have the following form:
if (condition) {
    statements;
} else if (condition) {
    statements;
} else {
    statements;
}